src/hooks/api/advertiser/p2p-advertiser/useAdvertiserInfo.ts

```diff
@@ -33,7 +33,7 @@ const useAdvertiserInfo = (id?: string) => {
     // Add additional information to the p2p_advertiser_info data
     useEffect(() => {
         if (data) {
-            const advertiser_info = data;
+            const advertiser_info = data.p2p_advertiser_info;
```

Comment:
can we remove the local storage values now?

Reply:
nope cant, we still need it

Reply:
but why?

Reply:
its the same reason as to why we needed it the last time, all the data we call from a hook will be scoped locally to the component itself. If i call subscribe from one place, i can't access it in another component, even for the children I couldn't access it. So we still need to store it in local storage. I only changed this line of code because the return type had changed